api/http/management.go

```diff
@@ -61,6 +62,32 @@ func (api *ManagementAPI) SetConfiguration(c *gin.Context) {
 		return
 	}
 
+	// RBAC
+	if len(c.Request.Header.Get(model.RBACHeaderDeploymentsGroups)) > 1 {
+		allowed := false
+		for _, group := range strings.Split(
+			c.Request.Header.Get(model.RBACHeaderDeploymentsGroups), ",") {
+			allowed, err = api.App.AreDevicesInGroup(
+				ctx, []string{devID.String()}, group)
+			if err != nil {
+				c.Error(err) //nolint:errcheck
+				rest.RenderError(c,
+					http.StatusInternalServerError,
+					errors.New(http.StatusText(http.StatusInternalServerError)),
+				)
+				return
+			}
+			if allowed {
+				break
+			}
+		}
+		if !allowed {
+			rest.RenderError(
+				c, http.StatusForbidden, errors.New("Access denied (RBAC)."))
+			return
+		}
+	}
+
```

Comment:
seems like a good candidate for a helper function.
can we have `ErrAccessDeniedByRBAC = errors.New("Access denied (RBAC).")` maybe?

Reply:
done